### DISCUSSION TITLE
The program is not responding

### DISCUSSION THREAD

**Discussion by mirth** [spacedriveapp/spacedrive#2029] [Community Support]
On the first program start I didn't gave full disk access to the Spacedrive.
Later on I gave full disk access to the Spacedrive (while it was on if it's important), then it stop responding - it doesn't show the window but the icon is in the dock. I've tried to restart the program but getting the same result.

The app is updated to the last suggested version, I suppose it's Alpha v0.2.2

**Comment by mirth** [comment]
Can someone tell me where is Spacedrive directory with configs? I'd like to delete it to fully reinstall the app.

**Reply by utkubakir** [reply]
`~/Library/Application\ Support/spacedrive`

**Reply by mirth** [reply]
I removed the directory but the just doesn't start.
I've also tried to toggle Full Disk Access.
May be I can provide logs somehow?

**Reply by utkubakir** [reply]
Can you try removing these also?

```
$HOME/Library/WebKit/Spacedrive
$HOME/Library/Caches/Spacedrive
$HOME/Library/WebKit/com.spacedrive.desktop
$HOME/Library/Caches/com.spacedrive.desktop
```

**Reply by mirth** [reply]
It didn't help.
There is app icon in dock but no app window.
